psalm.xml

```diff
@@ -10,6 +10,8 @@
             <directory name="vendor" />
             <directory name="src/Bundle/spec" />
             <directory name="src/Bundle/test" />
+            <file name="src/Bundle/DependencyInjection/SyliusMailerExtension.php" />
+            <file name="src/Bundle/Sender/Adapter/SwiftMailerAdapter.php" />
```

Comment:
Or maybe even have 2 different `psalm.xml` file, one for Symfony 5.4 and one for 6.0? Damn, supporting 2 versions with major bc-breaks between them is hard 😄

Reply:
It's not even the two Symfony majors here, it's SwiftmailerBundle not supporting anything newer than Symfony 5.4.  So you almost need a Psalm config for the core bundle + `symfony/mailer`, and an extended config for when Swift Mailer is installed.